modules/core/keeper/msg_server_test.go

```diff
@@ -888,7 +888,7 @@ func (suite *KeeperTestSuite) TestUpgradeClient() {
 		tc.setup()
 
 		ctx := suite.chainA.GetContext()
-		_, err = keeper.Keeper.UpgradeClient(*suite.chainA.App.GetIBCKeeper(), ctx, msg)
+		_, err = suite.chainA.GetSimApp().GetIBCKeeper().UpgradeClient(ctx, msg)
```

Comment:
decided to change this while I was here, makes no difference but kinda looks weird how it was 🧹 

I think these tests could be improved or cleaned up some more. But maybe outside the scope of this PR